app/registration/controllers/patientCommonController.js

```diff
@@ -32,13 +32,14 @@ angular.module('registration')
                                 $scope.patient.identifiers.push({identifierType: value,
                                     identifier: null, preferred: false,
                                     location: localStorageService.cookie.get("emr.location").uuid,
-                                    fieldName : fieldName});
+                                    fieldName : fieldName,
+                                    uimask: "(99999999/99/99999)",
```

Comment:
Hi @larslemos is that possible to get `uimask` from the identifier type metadata? This way we can guarantee consistency with RDE. Previously it was validating input based on the identifier type regex. For example NID (SERVICO TARV) is using `[0-9]{8}/[0-9]{2}/[0-9]{5}` of the field `Regex Format`. 
I think that we can use the field `Description of format` to pass the mask `99999999/99/99999`, or in a config file. This way we don't need to change the code if the mask change.